### PR TITLE
[docs]ncw playbook: minor improvements

### PR DESCRIPTION
**Pull request by tariq1890**
Minor improvements to the ncw playbook documentation. Fixed some dead links as well.

### CODE REVIEW

File: events/events-team/content/new-contributor-workshops/how-we-communicate.md [hunk 1]
```diff
@@ -10,11 +10,11 @@ Show how we communicate in the community, and which tools are for what purpose.
 
 ### Slides
 
-Refer to previous workshops for inspirations
+Refer to previous workshops for inspirations.
 
 ### Details To Cover 
 
-- Code of Conduct
+- Code of Conduct (CoC)
```
Comment (tpepper) [inline events/events-team/content/new-contributor-workshops/how-we-communicate.md]: Could add a link here too as you did with client-go and kind.

File: events/events-team/content/new-contributor-workshops/labels-and-bots.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ This task is meant as a short overview for Intermediate content.
 
 ### Slides
 
-The [PR workflow diagram](git.k8s.io/community/contributors/guide/git_workflow.png) is useful here. There are also previous slides that explain the differnet labels.
+The [PR workflow diagram](https://github.com/kubernetes/community/blob/master/contributors/guide/git_workflow.png) is useful here. There are also previous slides that explain the differnet labels.
```
Comment (tpepper) [inline events/events-team/content/new-contributor-workshops/labels-and-bots.md]: For git repo links we tend to prefer the "vanity url" redirector.  The links are shorter to read.

File: events/events-team/content/new-contributor-workshops/live-pr-demo.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Plan for about 30 minutes
 
 ### Task Overview
 
-Demonstrate opening a pull request against k/k. There are some docs cleanup good-first-issues, but feel free to show off something you've been working on for a while.
+Demonstrate opening a pull request against the Kubernetes/Kubernetes(k/k) repository. There are some docs cleanup good-first-issues, but feel free to show off something you've been working on for a while.
```
Comment (tpepper) [inline events/events-team/content/new-contributor-workshops/live-pr-demo.md]: Could add a link here to git.k8s.io/kubernetes aka github.com/kubernetes/kubernetes.  "k/k" is a frequently used piece of jargon likely unclear to a new contributor who might be reading this outline.

File: events/events-team/content/new-contributor-workshops/workspace-setup.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ Make sure all Beginning Contributors have a workspace setup for hacking on Kuber
 
 ### Slides
 
-Clear and large download commands to copy into terminal
+Clear and large download commands to copy onto the terminal.
```
Comment (tpepper) [inline events/events-team/content/new-contributor-workshops/workspace-setup.md]: I feel like "into the" would be proper here, but that's just a feeling.  Prepositions...the hardest part of most languages?

**Comment by tpepper** [comment]
Thanks for the cleaning up.  I've suggested a few potential additions while you're clarifying things.

**Comment by tariq1890** [comment]
@tpepper Thanks for the review! I've addressed the comments.

**Comment by mrbobbytables** [comment]
Thanks!
/approve

**Pull request merged**